Package.swift

```diff
@@ -14,9 +14,14 @@ let package = Package(
     ],
     dependencies: [],
     targets: [
-        .binaryTarget(
+        .target(
             name: "RadarSDK",
-            path: "dist/RadarSDK.xcframework"
+            path: "RadarSDK",
+            exclude: ["Info.plist", "RadarSDK.h"],
```

Comment:
Do we need to exclude these 2 things?  I don't see other peer (segment, one signal, amplitude) `Package.swift`'s making use of `exclude`.  Especially curious about `RadarSDK.h`, it seems strange that would need to be excluded.

Reply:
If I don't exclude `Info.plist` (which is used by `RadarSDK.xcodeproj`), then SPM 5.3+ (see the top line of `Package.swift`) complains about it being an unhandled file. Segment is using SPM 5.2, which doesn't complain, so they don't have to exclude the file. The `RadarSDK.h` exclusion may be a relic of when I was trying to get unit tests working in SPM; I'll check to see whether I still need it.

If I edit Segment's `Package.swift` to use SPM 5.3, then it complains, too. :)
<img width="266" alt="Screen Shot 2022-02-08 at 12 17 34 AM" src="https://user-images.githubusercontent.com/708510/152922968-e25b2ea6-246f-44b5-842f-b22f9d5dd9d2.png">

Reply:
👍